.jscsrc

```diff
@@ -1,5 +1,5 @@
 {
 	"preset": "jquery",
-
+	"maximumLineLength": 110,
```

Comment:
`100`, search the string "*Lines should be*" here https://contribute.jquery.org/style-guide/js/ :)